tests/shared/nlu/training_data/test_training_data.py

```diff
@@ -623,6 +629,40 @@ def test_custom_attributes(tmp_path):
     assert example.get("sentiment") == 0.8
 
 
+async def test_without_additional_e2e_examples(tmp_path: Path):
+    domain_path = tmp_path / "domain.yml"
+    domain_path.write_text(Domain.empty().as_yaml())
+
+    config_path = tmp_path / "config.yml"
+    config_path.touch()
+
+    existing = TrainingDataImporter.load_from_dict(
+        {}, str(config_path), str(domain_path), []
+    )
+
+    stories = StoryGraph(
+        [
+            StoryStep(
+                events=[
+                    UserUttered(None, {"name": "greet_from_stories"}),
+                    ActionExecuted("utter_greet_from_stories"),
+                ]
+            )
+        ]
+    )
+
+    # Patch to return our test stories
+    existing.get_stories = asyncio.coroutine(lambda *args: stories)
+
+    importer = E2EImporter(existing)
+
+    training_data = await importer.get_nlu_data()
+
+    assert training_data.training_examples
+    assert not training_data.is_empty()
+    assert len(training_data.nlu_examples) == 0
```

Comment:
I'd also check the exact content of the `nlu_examples` and verify if its correct.

Reply:
@joejuzl could you please take this over

Reply:
There is nothing in `training_data.nlu_examples` though, that's what this test is testing right?

Reply:
Ahh you are right. `nlu_examples` would be empty but `training_examples` won't be since we are adding names of `actions` and `intents` from the story to the `training_data`.

Reply:
Yes `training_data.training_examples` will also have all the default actions too. I think that behaviour is testing in `test_importer` though.